static/js/controllers/edit-contact.js

```diff
@@ -86,6 +92,10 @@ var libphonenumber = require('libphonenumber/utils'),
           };
           $scope.category = $scope.contact.type === 'person' ? 'person' : 'place';
           $scope.contactId = null;
+          if ($scope.category === 'place') {
+            // set the Primary Contact field for places to be 'blank'
+            $scope.contact.contact = $scope.parents[0];
```

Comment:
It wasn't immediately clear to me what $scope.parents[0] referred to. I would pull out the null contact as a variable and use it in both places so it can be named usefully.